Source/model/scene/Shape.swift

```diff
@@ -54,4 +54,5 @@ open class Shape: Node {
 
         return bounds
     }
+    
```

Comment:
It's better to avoid all these unnecessary changes.

Reply:
sorry( hard to track them when every file is refactored 5 times

Reply:
Just make a final review of all your changes before you commit something 😊

Reply:
of course I do. but final review in it client compares with previous commit. First I added a piece of code, and comparing to that commit, I removed a piece of code. Resulting extra line can't be seen at this point. However, I can see it after I commit something right here, which I shall do